packages/@aws-cdk/aws-lambda-nodejs/lib/package-manager.ts

```diff
@@ -26,7 +26,7 @@ export class PackageManager {
   public static PNPM = new PackageManager({
     lockFile: 'pnpm-lock.yaml',
     installCommand: ['pnpm', 'install'],
-    runCommand: ['pnpm', 'run'],
+    runCommand: ['pnpm', 'exec'],
```

Comment:
Getting `The "pnpm exec" command currently only works with the "-r" option` with only `exec`.

Reply:
Can you try:
```sh
pnpm --version

cd aws-cdk/packages/@aws-cdk/aws-lambda-nodejs

pnpm exec esbuild -- --bundle test/integ-handlers/ts-handler.ts
```

Reply:
OK it's working. For some reason not in the Docker image...